bridge/tsconfig.app.json

```diff
@@ -13,6 +13,7 @@
   ],
   "exclude": [
     "client/test.ts",
-    "client/**/*.spec.ts"
+    "client/**/*.spec.ts",
+    "client/app/_utils/test.utils.ts"
```

Comment:
Maybe we should also add `client/**/*.mock.ts` here

Reply:
done